### PR TITLE
Add code for parity to membersAdded method

### PR DESCRIPTION
**Pull request by matiasroldan6**
### Purpose
*What is the context of this pull request? Why is it being done?*

### Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp? (Include illustrative screenshots for context if applicable.)*

### Tests
*Is this covered by existing tests or new ones? If no, why not?*

### Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*

#### General
- [ ] I have commented my code, particularly in hard-to-understand areas	
- [ ] I have added or updated the appropriate tests	
- [ ] I have updated related documentation

### CODE REVIEW

**Review by Batta32 — CHANGES_REQUESTED** [review]
What about the VA? We need to apply the same changes, if not.. we can have duplicated messages using DLC too

File: templates/typescript/generator-bot-virtualassistant/generators/app/templates/sample-assistant/src/bots/_defaultActivityHandler.ts [hunk 1]
```diff
@@ -60,17 +61,23 @@ export class DefaultActivityHandler<T extends Dialog> extends TeamsActivityHandl
...
+                }
 
-        if (userProfile.name === undefined || userProfile.name.trim().length === 0) {
-            // Send new user intro card.
-            await turnContext.sendActivity(this.templateManager.generateActivityForLocale('NewUserIntroCard', userProfile));
-        } else {
-            // Send returning user intro card.
-            await turnContext.sendActivity(this.templateManager.generateActivityForLocale('ReturningUserIntroCard', userProfile));
+                await DialogEx.run(this.dialog, turnContext, this.dialogStateAccessor);
+            }
```
Comment (Batta32) [inline templates/typescript/generator-bot-virtualassistant/generators/app/templates/sample-assistant/src/bots/_defaultActivityHandler.ts:79]: We are missing the
```typescript
        // By calling next() you ensure that the next BotHandler is run.
        await next();
```

right?

File: templates/typescript/samples/sample-assistant/src/bots/defaultActivityHandler.ts [hunk 2]
```diff
@@ -60,17 +61,23 @@ export class DefaultActivityHandler<T extends Dialog> extends TeamsActivityHandl
...
+                }
 
-        if (userProfile.name === undefined || userProfile.name.trim().length === 0) {
-            // Send new user intro card.
-            await turnContext.sendActivity(this.templateManager.generateActivityForLocale('NewUserIntroCard', userProfile));
-        } else {
-            // Send returning user intro card.
-            await turnContext.sendActivity(this.templateManager.generateActivityForLocale('ReturningUserIntroCard', userProfile));
+                await DialogEx.run(this.dialog, turnContext, this.dialogStateAccessor);
+            }
```
Comment (Batta32) [inline templates/typescript/samples/sample-assistant/src/bots/defaultActivityHandler.ts:79]: We are missing the
```typescript
        // By calling next() you ensure that the next BotHandler is run.
        await next();
```

right?

**Review by Batta32 — CHANGES_REQUESTED** [review]
The Skill LGTM, the VA needs changes

File: templates/typescript/samples/sample-assistant/src/bots/defaultActivityHandler.ts (hunk 2 above)
Comment (matiasroldan6) [reply templates/typescript/samples/sample-assistant/src/bots/defaultActivityHandler.ts:79]: next doesn't exist in that method

Comment (Batta32) [reply templates/typescript/samples/sample-assistant/src/bots/defaultActivityHandler.ts:79]: Maybe we can add a parity in the `membersAdded` signature, to have parity between VA/Skill? What do you think?

Comment (matiasroldan6) [reply templates/typescript/samples/sample-assistant/src/bots/defaultActivityHandler.ts:79]: not sure what that does. If you ask me Idk if I would change the signature of the method just to make it like the skill's. I think that the parity should be most with the C# v.a. in any case. If we change the signature maybe we should re test all

Comment (Batta32) [reply templates/typescript/samples/sample-assistant/src/bots/defaultActivityHandler.ts:79]: As mentions in the command `// By calling next() you ensure that the next BotHandler is run.`, I don't think that we have another BotHandler to run.. so I don't think that we should re test all, if you want to validate it, go on. But I prefer to have a parity between TS VA - TS Skill too.

I mean, we must have parity between TS VA - TS Skill. As a reviewer if I review this, and notice a disparity between VA/Skill in the membersAdded, I will raise a flag probably.. and probably Peter will do the same

**Review by Batta32 — APPROVED** [review]
LGTM

**Pull request closed without merging**